Makefile

```diff
@@ -124,6 +124,9 @@ clone-kakadu: # Optionally, downloads Kakadu from its private git repo
 		echo "[INFO] Kakadu is not included in build because KAKADU_VERSION is not set." >&2; \
 	fi
 
+godoc:
```

Comment:
How about splitting `docs` (which just builds the docs) and `docs-site` (which runs the server too) that way we can use a regular docs build in CI to make sure there aren't any errors with it, or to generate docs to push to somewhere else.

Reply:
Oh, and you asked me what I was using, but apparently I didn't give you the full answer. Below is my Makefile:
```
docs: test # Generates documentation from the project's code
	go doc

docs-site: docs # Runs `pkgsite` to view the newly generated docs
	pkgsite -http=localhost:8080
```
I'd read that pkgsite was newer, had a better UI, had more features (better support for modules), etc.